0000-pre-shared-key-v2.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Pre-Shared Key V2 (PSKv2) Transport Protocol
+draft: 1
+---
+# Pre-Shared Key V2 (PSKv2) Transport Protocol
+
+This spec defines version 2 of the Pre-Shared Key Transport Protocol. PSKv2 implements end-to-end data encryption, condition and fulfillment generation, [End-to-End Quoting](#end-to-end-quoting) and [Chunked Payments](#chunked-payments). The approach to chunked payments is inspired by the Internet's Transmission Control Protocol (TCP).
```

Comment:
Can it also be used for test payments and/or stats collection?

Reply:
Sure it can be used for test payments, but would those be any different from normal payments or quotes? If not, I'm not sure it needs to be documented here